default_config.yaml

```diff
@@ -1,58 +1,162 @@
 ---
-#---#
-# Config for edx-video-worker
-#---#
+# ---
+# Database information
+# ---
 
-## AWS Storage
-aws_deliver_bucket:
-aws_storage_bucket:
-multi_upload_barrier: 2000000000
-boto_timeout: 60
+# SANDBOX
+#DATABASES:
+#    default:
+#        ENGINE: django.db.backends.sqlite3
+#        NAME: sandbox.db
+
+## PRODUCTION
+DATABASES:
+    default:
+        ENGINE: 'django.db.backends.mysql'
+        NAME:
+        USER:
+        PASSWORD:
+        HOST:
+        PORT: '3306'
+
+django_secret_key: ""
```

Comment:
@muzaffaryousaf I was trying to make a unified configuration file (in case you decide in the future to collapse worker and pipeline into one repo).